src/components/skills-quiz/tests/SelectJobCard.test.jsx

```diff
@@ -137,6 +138,25 @@ describe('<SelectJobCard />', () => {
     expect(screen.queryByText(TEST_UNIQUE_POSTINGS_2)).toBeInTheDocument();
   });
 
+  test('renders multiple job card without labor market data', () => {
+    const initialJobCardState = {
+      interestedJobs: [{
+        name: 'Engineer',
+        objectID: '11',
+        job_postings: [],
+      },
+      ],
+    };
+    render(
+      <SelectJobCardWithContext
+        initialAppState={initialAppState}
+        initialJobCardState={initialJobCardState}
+      />,
+    );
+    expect(screen.queryByText(initialJobCardState.interestedJobs[0].name)).toBeInTheDocument();
+    expect(screen.queryByText(SALARY_NOT_AVAILABLE)).toBeInTheDocument();
```

Comment:
nit: I think if we test the job postings not available case here then the coverage will pass